css-view-transitions-1/Overview.bs

```diff
@@ -987,83 +1019,14 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 			Issue: Define the algorithm used to clip the snapshot when it exceeds max size.
 	</div>
 
-## [=Animate a view transition=] ## {#animate-a-view-transition-algorithm}
+## [=Setup transition pseudo-elements=] ## {#setup-transition-pseudo-elements-algorithm}
```

Comment:
This algorithm used to be "create transition pseudo-elements", but I added the style rules for things that only need to be done once per transition, so I renamed it to reflect that.